### ISSUE TITLE
Disappearing axis text

### ISSUE THREAD

**Issue by AbcAeffchen** [goat1000/SVGGraph#31]
Sorry, me again.

If you draw a BarGarph like this

```
$barPlot = new SVGGraph(600, 400, ['force_assoc' => true]);
$barPlot->values([0 => 1, 2 => 3, 4 => 5]);
```

The `0` text does not show up on the x-axis. It is drawn correctly if `force_assoc` is false. It is also not drawn if `0` is in an other position than the first.

I looked into it but it is hard to track down where the text gets lost. I Could not find it yet, but I try to look into it during the weekend. But maybe you can find it faster.

Best,
Alex

**Comment by AbcAeffchen** [comment]
I found a fix for this issue. But somehow it feels like the wrong place since it seems to have something to do with the `force_assoc` option.

**Comment by goat1000** [comment]
Sorry, somehow I missed this issue - it looks like you found where the display is being prevented, but the source of the problem is the `Axis::getText()` function returning the (integer) key as the text value. 

Making `getText()` only return strings should fix this, but I'll have to do some testing.

**Comment by goat1000** [comment]
I've uploaded a fix in b0efb04 - it shouldn't have any bad side-effects.

**Comment by AbcAeffchen** [comment]
Thanks. This looks good. I couldn't see bad side effects and it fixed the bug.
